manifests/params.pp

```diff
@@ -1,30 +1,37 @@
 # aide::params sets the default values for parameters.
 class aide::params {
-  $package         = 'aide'
-  $mailto          = undef
-  $version         = 'latest'
-  $db_path         = '/var/lib/aide/aide.db'
-  $db_temp_path    = '/var/lib/aide/aide.db.new'
-  $gzip_dbout      = 'no'
-  $hour            = 0
-  $minute          = 0
-  $aide_log        = '/var/log/aide/aide.log'
-  $syslogout       = true
-  $config_template = 'aide/aide.conf.erb'
-  $cron_template   = 'aide/cron.erb'
-  $nocheck         = false
+  $package              = 'aide'
+  $mailto               = undef
+  $mail_only_on_changes = false
+  $version              = 'latest'
+  $db_path              = '/var/lib/aide/aide.db'
+  $db_temp_path         = '/var/lib/aide/aide.db.new'
+  $gzip_dbout           = 'no'
+  $hour                 = 0
+  $minute               = 0
+  $aide_log             = '/var/log/aide/aide.log'
+  $syslogout            = true
+  $config_template      = 'aide/aide.conf.erb'
+  $cron_template        = 'aide/cron.erb'
+  $nocheck              = false
 
   case $::osfamily {
     'Debian': {
       $aide_path = '/usr/bin/aide'
+      $cat_path  = '/bin/cat'
+      $rm_path   = '/bin/rm'
       $conf_path = '/etc/aide/aide.conf'
     }
     'Redhat': {
       $aide_path = '/usr/sbin/aide'
+      $cat_path  = '/usr/bin/cat'
+      $rm_path   = '/usr/bin/rm'
```

Comment:
Pretty sure `/bin/rm` would be correct for RedHat too.

Reply:
I think if its RHEL 6 only ```/bin/rm``` is available. On RHEL 7 both ```/bin/rm``` and ```/usr/bin/rm``` are available.